### DISCUSSION TITLE
Random small triangles in lazer osu!cookie

### DISCUSSION THREAD

**Discussion by Vorondanil54** [ppy/osu#23134] [Ideas]
So, while playing lazer I've noticed random ultrasmall triangles appearing in osu!cookie, idk what but devs must fix this
![osu_2023-04-04_16-44-51](https://user-images.githubusercontent.com/62338698/229983725-a39257b6-70bf-4ba1-934e-456d18be86d0.jpg)

**Comment by Rekunan** [comment]
I'd assume that's intentional, since all the triangles in the cookie are randomized.

**Reply by TheFearlessDeath500** [reply]
Is it random in a specific range?

**Reply by Rekunan** [reply]
No idea, I'd have to look in the code for that, which I can't do rn.
